girder/api/v1/resource.py

```diff
@@ -180,7 +180,15 @@ def _lookUpToken(self, token, parentType, parent):
         raise RestException('Child resource not found: %s(%s)->%s' % (
             parentType, parent.get('name', parent.get('_id')), token))
 
-    def _lookUpPath(self, path, test, user):
+    def _lookUpPath(self, path, user, test=False):
+        """
+        Look up a resource in the data hierarchy by path.
+        :param path: path of the resource
+        :param user: user with correct privledges to access path
```

Comment:
Also add a blank line between the description and the param list so that sphinx-build doesn't complain.